python/tank/platform/engine.py

```diff
@@ -192,6 +194,10 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt_abstraction.QtCore = qt.QtCore
         qt_abstraction.QtGui = qt.QtGui
 
+        # load the fonts. this will work if there is a QApplication instance
+        # available.
+        self._ensure_core_fonts_loaded()
```

Comment:
Could this be private, e.g. with a double underscore? Or are we expecting it to be called from subclasses of the engine?